lib/tasks/employees.rake

```diff
@@ -0,0 +1,41 @@
+namespace :employees do
+  desc "Imports all employees from Slack into the Dolores Landingham bot."
+  task import: :environment do
+    logger = ActiveSupport::Logger.new("log/employees_import.log")
+    start_time = Time.now
```

Comment:
This is also a good use for the `Benchmark.realtime` method, although that would require defining the `import_statistics` variable outside the block first, so I guess it's more of a tossup